src/model.jl

```diff
@@ -527,19 +728,20 @@ function _build_model(
     storage_dis = storage_enabled ? storage_dis : nothing
     storage_chg = storage_enabled ? storage_chg : nothing
     storage_soc = storage_enabled ? storage_soc : nothing
-    initial_soc = storage_enabled ? initial_soc : nothing
-    initial_rampup = initial_ramp_enabled ? initial_rampup : nothing
-    initial_rampdown = initial_ramp_enabled ? initial_rampdown : nothing
-    load_shed_ub = load_shed_enabled ? load_shed_ub : nothing
+    initial_soc = storage_enabled ? m[:initial_soc] : nothing
+    initial_rampup = initial_ramp_enabled ? m[:initial_rampup] : nothing
+    initial_rampdown = initial_ramp_enabled ? m[:initial_rampdown] : nothing
+    load_shed_ub = load_shed_enabled ? m[:load_shed_ub] : nothing
     voi = VariablesOfInterest(;
```

Comment:
I know we use the `VariablesOfInterest` struct in both `loop.jl` and `query.jl`, but I'm curious if using the syntax that was mentioned at the beginning of the PR might work too (i.e., calling `m[:foo]` for variable/constraint `foo` whenever needed). Since the model is already present, we would be accessing something that is already stored in memory. Do you know if this would work or if there would be any benefit in doing this over the current method?

Reply:
That might work! The more I look at `VariablesOfInterest`, the less I like it. It would definitely be cleaner code to just look directly into the model itself, now that we know the convenient syntax to do so.

Reply:
This is implemented in the latest batch of commits, and _seems_ to work...

Reply:
Looks great!